sdcm/tester.py

```diff
@@ -865,19 +866,21 @@ def set_system_auth_rf(self, db_cluster=None):
             # TODO: move this skip to siren-tools when possible
             self.log.warning("Skipping this function due this job run from Siren cloud!")
             return
-        # change RF of system_auth
-        system_auth_rf = self.params.get('system_auth_rf')
```

Comment:
If we aren't using this one anymore, we should remove it from all configuration files (and from sct_config.py)

Reply:
removed